cms/envs/common.py

```diff
@@ -228,14 +228,9 @@
 for namespace, template_dirs in lms.envs.common.MAKO_TEMPLATES.iteritems():
     MAKO_TEMPLATES['lms.' + namespace] = template_dirs
 
+# Django templating
 TEMPLATE_DIRS = MAKO_TEMPLATES['main']
```

Comment:
@nedbat : I agree with you - there's even a Django warning that states that. However, I spent a few hours attempting it this morning and have failed so far. I've written a ticket to do the work - it's here:

https://openedx.atlassian.net/browse/TNL-3532

I plan to move on for now. Once this PR is merged, I'll check in my non-functional work into a branch and add it to the ticket.